CONTRIBUTING.md

```diff
@@ -0,0 +1,167 @@
+
+* [![Bugs](https://img.shields.io/badge/-bugs-fc2929.svg)][link_bugs] *These issues point to problems in the project.*
+
+    If you find new a bug, please give as much detail as possible in your issue, including steps to recreate the error.
+    If you experience the same bug as one already listed, please add any additional information that you have as a comment.
+
+* [![Enhancement](https://img.shields.io/badge/-enhancement-84b6eb.svg)][link_enhancement] *These issues are asking for enhancements to be added to the project.*
+
+    Please try to make sure that your enhancement is distinct from any others that have already been requested or implemented.
+    If you find one that's similar but there are subtle differences please reference the other request in your issue.
+
+## Making a change
+
+We appreciate all contributions to `jupyter-book`, but those accepted fastest will follow a workflow similar to the following:
+
+**1. Comment on an existing issue or open a new issue referencing your addition.**
+
+This allows other members of the jupyter-book development team to confirm that you aren't overlapping with work that's currently underway and that everyone is on the same page with the goal of the work you're going to carry out.
+
+[This blog][link_pushpullblog] is a nice explanation of why putting this work in up front is so useful to everyone involved.
+
+**2. [Fork][link_fork] the [jupyter-book repository][link_jupyter-book] to your profile.**
+
+This is now your own unique copy of jupyter-book.
+Changes here won't effect anyone else's work, so it's a safe space to explore edits to the code!
+
+Make sure to [keep your fork up to date][link_updateupstreamwiki] with the master repository.
+
+**3. Make the changes you've discussed.**
+
+Try to keep the changes focused.
+We've found that working on a [new branch][link_branches] makes it easier to keep your changes targeted.
+
+**4. Submit a [pull request][link_pullrequest].**
+
+A member of the development team will review your changes to confirm that they can be merged into the main code base.
+When opening the pull request, we ask that you follow some [specific conventions](#pull-requests).
+We outline these below.
+
+### Pull Requests
+
+To improve understanding pull requests "at a glance", we encourage the use of several standardized tags.
+When opening a pull request, please use at least one of the following prefixes:
+
+* **[BRK]** for changes which break existing builds or tests
+* **[DOC]** for new or updated documentation
+* **[ENH]** for enhancements
+* **[FIX]** for bug fixes
+* **[REF]** for refactoring existing code
+* **[STY]** for stylistic changes
+* **[TST]** for new or updated tests, and
+* **[WIP]** for changes which are not yet ready to be merged
+
+Pull requests should be submitted early and often!
+If your pull request is not yet ready to be merged, please also include the **[WIP]** prefix.
```

Comment:
For this one maybe we should start instructing folks to use the github "draft PR" feature?

Reply:
I still have confusion around its uses myself, clearly ! :laughing: 

But I can definitely update these guidelines to use it which will hopefully resolve some(one else's) ambiguity.